examples/pytorch/text-generator/README.md

```diff
@@ -150,148 +142,31 @@ local   text-generator   live     1            1           17m           3.1285
 
 The output above indicates that one replica of your API was requested and is available to serve predictions. Cortex will automatically launch more replicas if the load increases and will spin down replicas if there is unused capacity.
 
-Show additional information for your API (e.g. its endpoint) using `cortex get <api_name>`:
-
-```bash
-$ cortex get text-generator --env aws
-
-status   up-to-date   requested   last update   avg request   2XX
-live     1            1           17m           -             -
-
-metrics dashboard: https://us-west-2.console.aws.amazon.com/cloudwatch/home#dashboards:name=cortex
-endpoint: https://***.execute-api.us-west-2.amazonaws.com/text-generator
-...
-```
```

Comment:
Consider bringing back this section, since I think it is useful (some people don't realize you can do `cortex get <api_name>` to get the endpoint)

Reply:
Edit: I see that we show `cortex get <api_name>` in the deploying locally section, so I don't feel as strongly about including it here. I still slightly prefer it, since it's an "easy win" (low friction command, good to remind you of it).